cpu/lm4f120/periph/timer.c

```diff
@@ -14,6 +15,7 @@
  * @brief       Implementation of the low-level timer driver for the LM4F120
  *
  * @author      Rakendra Thapa <rakendrathapa@gmail.com>
+ *              Marc Poulhiès <dkm@kataplop.net>
  */
 
 #include <stdint.h>
```

Comment:
Concerning the DEBUG, it's true that none survived, but that was not really intended. I think keeping the armless include+define DEBUG is good idea. Future use of DEBUG() will work.
Ok for the 2nd remark.
I'll repush the branch (already rebased locally, needs to publish it)